modules/cassandra/testcontainers/cassandra/__init__.py

```diff
@@ -0,0 +1,53 @@
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import os
+from typing import Optional
+
+from cassandra.cluster import Cluster, NoHostAvailable
+
+from testcontainers.core.container import DockerContainer
+from testcontainers.core.waiting_utils import wait_container_is_ready, wait_for_logs
+
+
+class CassandraContainer(DockerContainer):
+    """
+    Cassandra database container.
+
+    Example:
+
+        .. doctest::
+
+            >>> from testcontainers.cassandra import CassandraContainer
+            >>> from cassandra.cluster import Cluster
+
+            >>> with CassandraContainer("cassandra:latest") as cassandra:
+            ...    cluster = Cluster(
+            ....                 [cassandra.get_container_host_ip()],
+            ...                  port=cassandra.get_exposed_port(cassandra.port)
+            ...              )
+            ...    session = cluster.connect()
+            ...    result = session.execute("SELECT release_version FROM system.local;")
+    """
+
+    def __init__(self, image: str = "cassandra:latest", port: int = 9042, **kwargs) -> None:
+        super().__init__(image=image, **kwargs)
+        self.port = port
+        self.with_exposed_ports(self.port)
+
+    def _connect(self):
+        wait_for_logs(self, "Startup complete")
```

Comment:
What is the preferred strategy for the readiness probe ?
I have used `wait_for_logs` here because it doesn't need to depend on the cassandra driver.
But I see that the Java implementation [uses the driver](https://github.com/testcontainers/testcontainers-java/blob/main/modules/cassandra/src/main/java/org/testcontainers/containers/wait/CassandraQueryWaitStrategy.java#L33) and it gives the opportunity to [provide an init script](https://github.com/testcontainers/testcontainers-java/blob/main/modules/cassandra/src/main/java/org/testcontainers/containers/CassandraContainer.java#L154).